readme-docs/RELEASING.md

```diff
@@ -32,7 +32,7 @@ The steps are identical for both release types, unless specified otherwise below
 - Merge the PRs
 
 ### Always
-- Make sure there's a section in [upgrading.md](https://documentation.defectdojo.com/dev/getting_started/upgrading/) about any specific instructions when upgrading to this new release.
+- Make sure there's a section in [upgrading.md](../docs/content/en/open_source/upgrading) about any specific instructions when upgrading to this new release.
```

Comment:
Same here with relative link where everywhere else they are absolute. Just wanted to make sure this is the expectation.

Reply:
yes, this way users will land on the actual md file they need to edit